pkg/channel/distributed/controller/kafkachannel/dispatcher.go

```diff
@@ -300,6 +301,37 @@ func (r *Reconciler) reconcileDispatcherDeployment(ctx context.Context, logger *
 
 		// Log Deletion Timestamp & Finalizer State
 		if deployment.DeletionTimestamp.IsZero() {
+			deploymentCopy := deployment.DeepCopy()
+			needsUpdate := false
+
+			if deploymentCopy.Spec.Template.Annotations == nil {
+				logging.FromContext(ctx).Infof("Configmap hash is not set. Updating the dispatcher deployment.")
```

Comment:
This is probably because of grabbing code from the consolidated side, but we already have the 'logger' variable available in this function, so re-extracting it from the context here(and a couple of other places; won't comment on them all) isn't necessary.

Reply:
Yeah, copy-paster error. Pushed a new commit now.